tests/conftest.py

```diff
@@ -64,10 +75,13 @@ def boto_config(request):
 
 @pytest.fixture(scope='session')
 def sagemaker_session(sagemaker_client_config, sagemaker_runtime_config, boto_config):
-    boto_session = boto3.Session(**boto_config) if boto_config else boto3.Session(region_name=DEFAULT_REGION)
+    boto_session = boto3.Session(**boto_config) if boto_config else boto3.Session(
+        region_name=DEFAULT_REGION)
```

Comment:
not in scope of this PR, but seeing this and scanning through [conftest.py](https://github.com/aws/sagemaker-python-sdk/blob/master/tests/conftest.py) makes me think we should make a `boto_session` fixture

Reply:
agree, there should be another PR for this.